### PR TITLE
Test integration

### PR DESCRIPTION
**Pull request by prantik1998**

### CODE REVIEW

File: src/api/labApi/userListApi.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const userListApi = {
 }
 
 const token = labstate()['token'];
+console.log(localStorage.getItem("full-user-info"))
```
[the remark this answers is not in the document]
Comment (Liza23) [reply src/api/labApi/userListApi.js:11]: Now, it is not needed, I've saved the user and lab information in the constants file.

[the remark this answers is not in the document]
Comment (Dheeraj-DJP) [reply src/api/labApi/userListApi.js:11]: Has that been merged to master or pushed to some branch?

**Review by Liza23 — COMMENTED** [review]
Once I merge the lab-id constants file PR, then you need to change how you're calling `user-info` right now.

**Comment by vyasakanksha** [comment]
@Liza23 we need to merge the lab-id-constants PR urgently. Can we quickly discuss it?

**Review by Dheeraj-DJP — APPROVED** [review]
Looks good. Getting labId from constant file, I will add in next PR along with other modifying labId in other views.

**Comment by Dheeraj-DJP** [comment]
While resolving conflicts, please keep the content from master branch in the two files in src/ folder.

**Pull request merged**